dspace-api/src/main/java/org/dspace/app/statistics/clarin/ClarinMatomoBitstreamTracker.java

```diff
@@ -0,0 +1,135 @@
+/**
+ * The contents of this file are subject to the license and copyright
+ * detailed in the LICENSE and NOTICE files at the root of the source
+ * tree and available online at
+ *
+ * http://www.dspace.org/license/
+ */
+package org.dspace.app.statistics.clarin;
+
+import java.sql.SQLException;
+import java.util.List;
+import java.util.Objects;
+import javax.servlet.http.HttpServletRequest;
+
+import org.apache.commons.collections4.CollectionUtils;
+import org.apache.commons.lang3.BooleanUtils;
+import org.apache.commons.lang3.StringUtils;
+import org.apache.log4j.Logger;
+import org.dspace.content.Bitstream;
+import org.dspace.content.Item;
+import org.dspace.content.MetadataValue;
+import org.dspace.content.service.ItemService;
+import org.dspace.content.service.clarin.ClarinItemService;
+import org.dspace.core.Context;
+import org.dspace.services.ConfigurationService;
+import org.dspace.services.factory.DSpaceServicesFactory;
+import org.matomo.java.tracking.CustomVariable;
+import org.matomo.java.tracking.MatomoException;
+import org.matomo.java.tracking.MatomoRequest;
+import org.springframework.beans.factory.annotation.Autowired;
+
+/**
+ * Customized implementation of the ClarinMatomoTracker for the tracking the Item's bitstream downloading events
+ *
+ * The class is copied from UFAL/CLARIN-DSPACE (https://github.com/ufal/clarin-dspace) and modified by
+ * @author Milan Majchrak (milan.majchrak at dataquest.sk)
+ */
+public class ClarinMatomoBitstreamTracker extends ClarinMatomoTracker {
```

Comment:
Is it still possible to configure tracking per collection (enable/disable; different site id). Originally this was added in https://github.com/ufal/clarin-dspace/pull/886